src/content_script.js

```diff
@@ -845,46 +909,15 @@ async function CheckIfURLShouldBeBlocked() {
   if (siteList.includes(site)) {
     checkForTrackers = false;
   } else {
-    // Initialize the content script the first time
     await contentScriptInit(false);
-
-    // Reinitialize the content script for mutated nodes
-    const observer = new MutationObserver((mutations) => {
-      new Set(mutations.flatMap(mutation => {
-        switch (mutation.type) {
-        case "attributes":
-          return mutation.target;
-        case "childList":
-          return Array.from(mutation.addedNodes);
-        default:
-          return [];
-        }
-      })).forEach(target => contentScriptInit(false, null, target));
-    });
-
-    // Check for mutations in the entire document
-    observer.observe(document, {
-      childList: true,
-      attributes: true,
-      attributeFilter: OBSERVER_ATTRIBUTES,
-      subtree: true
-    });
```

Comment:
This is also a rebase artifact. (https://github.com/mozilla/contain-facebook/pull/736)

Reply:
Added back 13b9a16.